src/schema/v2/auction_result.ts

```diff
@@ -240,4 +242,36 @@ const AuctionResultType = new GraphQLObjectType<any, ResolverContext>({
 
 export const auctionResultConnection = connectionWithCursorInfo({
   nodeType: AuctionResultType,
+  connectionFields: {
+    earliestCreatedYear: {
```

Comment:
Just musing, `auctionCreatedYearRangeLoader` appears twice in this file (once per each of these fields). Therefore, it seems like it's _possible_ that it's requested twice (altho should be deduped properly by the data loader), but really, it seems like these fields only make sense together (and should share the data).

Maybe a connection field like:
```
dateRange: {
  earliestCreatedYear: ...,
  latestCreatedYear: ...
}
``` 
can better express that, and also will naturally have one resolver that needs to fetch those dates?

**the above is purely musing b/c I was tagged, it's otherwise pretty minor, there is def. no one answer**

Reply:
That's a very valid point. I'll give this some consideration before merging to avoid having to make a breaking change later. 

it could be something like

```
createdYearRange: {
	earliestCreatedYear: ...
	latestCreatedYear: ...
}
```

or something...